### PR TITLE
#62: HasLines doesn't explain the failed scenario properly

### PR DESCRIPTION
**Pull request by dgroup**
#62

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#63](https://github.com/llorllale/cactoos-matchers/issues/63) is now in scope, role is `REV`

File: src/main/java/org/llorllale/cactoos/matchers/HasLines.java [hunk 1]
```diff
@@ -100,22 +99,29 @@ public HasLines(
         final Collection<String> lns
     ) {
         super();
-        this.fnc = new UncheckedBiFunc<>(fnc);
-        this.separator = new UncheckedScalar<>(stor);
```
Comment (llorllale) [inline src/main/java/org/llorllale/cactoos/matchers/HasLines.java]: @dgroup can you do me a favor and rename `stor` to something like `sep`?

**Comment by llorllale** [comment]
@dgroup just one minor comment, please check

File: src/main/java/org/llorllale/cactoos/matchers/HasLines.java (hunk 1 above)
Comment (dgroup) [reply src/main/java/org/llorllale/cactoos/matchers/HasLines.java]: @llorllale sure :)

**Comment by dgroup** [comment]
@llorllale, done.

**Comment by llorllale** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:llorllale/cactoos-matchers#63` is not assigned, can't get performer

**Comment by 0crat** [comment]
The job [#63](https://github.com/llorllale/cactoos-matchers/issues/63) is now out of scope

**Pull request merged**